FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -113,12 +113,15 @@ extension EncryptedStorage: LogOutHandler {
                 .filter { keys.map(\.longid).contains($0.longid) }
             let sessions = storage.objects(SessionObject.self)
                 .filter { $0.email == email }
+            let clientConfigurations = storage.objects(ClientConfigurationObject.self)
+                .filter { $0.userEmail == email }
 
             try storage.write {
                 storage.delete(keys)
                 storage.delete(sessions)
                 storage.delete(passPhrases)
                 storage.delete(userToDelete)
+                storage.delete(clientConfigurations)
```

Comment:
@Kharchevskyi does the order of items matter here? If there are some foreign keys then we probably need to delete the user last?

Reply:
I think in this case - yes. 
ClientConfigurationObject has userObject as a reference. 
To be sure please check if this cause a crash on log out when there are 2 accounts logged in.